src/Kernel.php

```diff
@@ -10,4 +10,10 @@
 class Kernel extends BaseKernel
 {
     use MicroKernelTrait;
+
+    public function boot(): void
+    {
+        parent::boot();
+        date_default_timezone_set('UTC');
```

Comment:
Ici aussi, tu pourrais passer par un paramètre dynamique (tu as un exemple ici : https://lindevs.com/set-default-time-zone-in-symfony